test/firefox/test_profile_skeleton/prefs.js

```diff
@@ -1,20 +1,9 @@
-// Submit SSL Observatory reports.
-user_pref("extensions.https_everywhere._observatory.enabled", true);
-// Don't show any popups that might get in the way of testing.
-user_pref("extensions.https_everywhere._observatory.popup_shown", true);
-user_pref("extensions.https_everywhere.toolbar_hint_shown", true);
-// Show all logs.
-user_pref("extensions.https_everywhere.LogLevel", 0);
-user_pref("extensions.https_everywhere.log_to_stdout", true);
 // Make it quicker to make manual config changes.
```

Comment:
These changes belong in https://github.com/EFForg/https-everywhere/pull/11791, I'll move them over.